madgraph5amcatnlo.spec

```diff
@@ -0,0 +1,32 @@
+### RPM external madgraph5_amcatnlo 2.3.0.beta
```

Comment:
Please change the name of the package to madgraph5amcatnlo (no underscore) as the spec and as it's Required in the toolfile.